litmus-portal/cluster-k8s-manifest.yml

```diff
@@ -100,24 +68,11 @@ metadata:
     name: litmus-server
 rules:
   - apiGroups:
-      - ""
-    resources:
-      - pods
-      - pods/exec
-      - services
-      - nodes
-    verbs:
-      - create
-      - patch
-      - update
-      - get
-      - list
-  - apiGroups:
-      - "apps"
+      - "*"
     resources:
-      - deployments
+      - "*"
     verbs:
-      - create
+      - "*"
```

Comment:
is it not working without all GRVs?

Reply:
We should restrict the permissions needed